From 78bc7bdb4f616a601090a98c2b974233fab6bd09 Mon Sep 17 00:00:00 2001
From: Huacai Chen <chenhc@lemote.com>
Date: Thu, 1 Dec 2016 09:51:35 +0800
Subject: [PATCH 067/137] Change some KERN_ERR to KERN_INFO

1, In radeon graphics card.
2, In i8042.c, SB710 has no i8042 controller so selftest timeout.

Signed-off-by: Huacai Chen <chenhc@lemote.com>
---
 drivers/gpu/drm/radeon/radeon_atombios.c | 2 +-
 drivers/gpu/drm/radeon/rs690.c           | 2 +-
 drivers/input/serio/i8042.c              | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/radeon/radeon_atombios.c b/drivers/gpu/drm/radeon/radeon_atombios.c
index 9e0aa357585f..ad7dd4fafcf4 100644
--- a/drivers/gpu/drm/radeon/radeon_atombios.c
+++ b/drivers/gpu/drm/radeon/radeon_atombios.c
@@ -1331,7 +1331,7 @@ bool radeon_atombios_sideport_present(struct radeon_device *rdev)
 				return true;
 			break;
 		default:
-			DRM_ERROR("Unsupported IGP table: %d %d\n", frev, crev);
+			DRM_INFO("Unsupported IGP table: %d %d\n", frev, crev);
 			break;
 		}
 	}
diff --git a/drivers/gpu/drm/radeon/rs690.c b/drivers/gpu/drm/radeon/rs690.c
index 267d8a9134c8..b66a23ffaa2e 100644
--- a/drivers/gpu/drm/radeon/rs690.c
+++ b/drivers/gpu/drm/radeon/rs690.c
@@ -112,7 +112,7 @@ void rs690_pm_info(struct radeon_device *rdev)
 			rdev->pm.igp_system_mclk.full = dfixed_const(200);
 			rdev->pm.igp_ht_link_clk.full = dfixed_const(1000);
 			rdev->pm.igp_ht_link_width.full = dfixed_const(8);
-			DRM_ERROR("No integrated system info for your GPU, using safe default\n");
+			DRM_INFO("No integrated system info for your GPU, using safe default\n");
 			break;
 		}
 	} else {
diff --git a/drivers/input/serio/i8042.c b/drivers/input/serio/i8042.c
index 168c8854a7a6..c942336332e0 100644
--- a/drivers/input/serio/i8042.c
+++ b/drivers/input/serio/i8042.c
@@ -953,7 +953,7 @@ static int i8042_controller_selftest(void)
 	do {
 
 		if (i8042_command(&param, I8042_CMD_CTL_TEST)) {
-			pr_err("i8042 controller selftest timeout\n");
+			pr_info("i8042 controller selftest timeout\n");
 			return -ENODEV;
 		}
 
-- 
2.37.1

